WolvenKit.Common/Enums.cs

```diff
@@ -31,6 +31,7 @@ public enum EFileReadErrorCodes
     /// </summary>
     public enum ECookedFileFormat
     {
+        mesh,
```

Comment:
This enum should only contain extensions for which uncooking (ie. converting to a raw file format) has already been implemented. For all other cases, simply extract the buffers.